e2e/vmss.go

```diff
@@ -459,8 +471,12 @@ func getBaseVMSSModel(s *Scenario, customData, cseCmd string) armcompute.Virtual
 				StorageProfile: &armcompute.VirtualMachineScaleSetStorageProfile{
 					OSDisk: &armcompute.VirtualMachineScaleSetOSDisk{
 						CreateOption: to.Ptr(armcompute.DiskCreateOptionTypesFromImage),
-						DiskSizeGB:   to.Ptr(int32(512)),
+						DiskSizeGB:   to.Ptr(int32(50)),
 						OSType:       to.Ptr(armcompute.OperatingSystemTypesLinux),
+						Caching:      to.Ptr(armcompute.CachingTypesReadOnly),
+						DiffDiskSettings: &armcompute.DiffDiskSettings{
+							Option: to.Ptr(armcompute.DiffDiskOptionsLocal),
```

Comment:
what is this option for exactly ? I can't seem to find a clear description for it.

Reply:
it's ephemeral disks